packages/stylelint-config/README.md

```diff
@@ -19,27 +19,113 @@ npm install --save-dev @lmc-eu/stylelint-config stylelint-config-prettier
 > We assume you are using Prettier. That's why we also recommend adding
 > `stylelint-config-prettier` above.
 
-## Configurations
+New configuration extends community maintained config [stylelint-config-standard-scss](https://github.com/stylelint-scss/stylelint-config-standard-scss).
```

Comment:
I would suggest avoiding the word _new_. It is new now, but this sentence will outdate soon.

Reply:
True true, fixed.